tests/pipelines/test_cli.py

```diff
@@ -273,19 +273,19 @@ def test_allow_duplicate_names_flag(pipelines_api_mock, tmpdir):
         f.write(DEPLOY_SPEC_NO_ID)
     runner = CliRunner()
     runner.invoke(cli.deploy_cli, [path])
-    assert pipelines_api_mock.create.call_args_list[0][0][1] is False
+    assert pipelines_api_mock.create.call_args_list[0][0][2] is False
```

Comment:
These test that the `allow_duplicate_names` parameter of `create` and `deploy` is set correctly. Because I added a new parameter `spec_dir`, this shifts by one from position 1 to 2.